src/components/App/App.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import TodoList from "../TodoComponents/TodoList/TodoList";
+import TodoForm from "../TodoComponents/TodoForm/TodoForm";
+import "./App.css";
+import activityList from "./activityList";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.todoList = null;
+
+    try {
+      this.todoList = JSON.parse(localStorage.taskManager);
+    } catch (e) {
+      this.todoList = activityList;
+    }
+
+    this.state = {
+      todos: this.todoList,
+      newTodoInput: "",
+      searchText: ""
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener("beforeunload", () => {
+      this.updateLocalStorage();
```

Comment:
Good job on storing data to local storage to make the page persistent to reloads.